Automattic/wp-calypso
# Split `client/lib/analytics/utils` into multiple modules.

Instead of one monolithic file that can't easily be tree-shaken, this change splits `client/lib/analytics/utils` into individual modules.

This will not have any immediate impact (in fact, it slightly increases the overall build size by around 300 bytes [compressed], due to the extra module plumbing needed), but should come into play with e.g. #33642. 

#### Changes proposed in this Pull Request

* Split `client/lib/analytics/utils` into individual modules for each exported unit.

#### Testing instructions

No specific testing instructions. This PR shouldn't change any behaviour, only move some code around. E2E tests should help ensure sanity.
